server/controllers/refer/new/referralsController.ts

```diff
@@ -97,6 +97,13 @@ export default class NewReferralsController {
       const { courseOfferingId, prisonNumber } = req.body
       const { username, userId } = req.user
 
+      const course = await this.courseService.getCourseByOffering(req.user.username, courseOfferingId)
+      if (!course.referable) {
+        throw createError(400, {
+          userMessage: 'Course is not referable.',
+        })
```

Comment:
This should be `400, 'Course is not referable'` - then I think we can test the message too